src/core/libmaven/mzUtils.cpp

```diff
@@ -392,12 +392,31 @@ Series:  Prentice-Hall Series in Automatic Computation
         return(str);
     }
 
-    float ppmDist(const float mz1, const float mz2) {
-        return ( abs((mz2-mz1)/(mz1/1e6)) );
+    float massCutoffDist(const float mz1, const float mz2,MassCutoff *massCutoff) {
+        if(massCutoff->getMassCutoffType()=="ppm"){
+            return ( abs((mz2-mz1)/(mz1/1e6)) );
+        }
+        else if(massCutoff->getMassCutoffType()=="mDa"){
+            return abs((mz2-mz1)*1e3) ;
```

Comment:
Any specific reason for 10^3?

Reply:
Shubhra suggested it.

Reply:
Da is equal to amu. MiliDalton is 10^-3 amu. Correct me if I'm wrong though.